src/pages/about.astro

```diff
@@ -16,43 +16,50 @@ import FooterFull from '../components/footer/FooterFull.svelte'
     <img src="/ega.svg" />
   </div>
   <div
-    class="relative max-h-screen overflow-y-scroll font-expressway red-radial-gradient scrollbar-hide snap-start snap-y snap-mandatory"
+    class="relative max-h-screen overflow-y-scroll font-expressway red-radial-gradient scrollbar-hide"
   >
     <div
-      class="relative z-20 h-screen py-20 snap-start bg-[url('/images/marine.png')] bg-cover bg-no-repeat"
+      class="relative z-20 h-screen py-20 bg-[url('/images/marine.png')] bg-cover bg-no-repeat"
     >
       <Hero
         padding="pt-24"
         title="WHO WE ARE"
         content="The Marine Corps Software Factory welcomes Marines from any MOS who have the aptitude, competence, and discipline to serve in key roles on software development teams."
+        contentClasses="text-2xl"
```

Comment:
This breaks the mobile font size, and you modified Hero to set this size anyway. Either remove entirely and have the default hero font size be set or use md:text-2xl.

Reply:
fixed thanks